torch/csrc/distributed/rpc/init.cpp

```diff
@@ -448,8 +448,8 @@ If the future completes with an error, an exception is thrown.
       "_invoke_remote_torchscript",
       [](const std::string& dstWorkerName,
          const std::string& qualifiedNameStr,
-         const py::args& args,
-         const py::kwargs& kwargs) {
+         py::args args,
+         py::kwargs kwargs) {
```

Comment:
Shall add a NOLINT to get rid of lint failure and add some comments to explain?

Reply:
Yes. Updated.